env.sh

```diff
@@ -0,0 +1,6 @@
+{
+  echo REACT_APP_VERCEL_GIT_REPO_OWNER=compose-run
+  echo REACT_APP_VERCEL_GIT_REPO_SLUG=community 
+  echo REACT_APP_VERCEL_GIT_COMMIT_REF=$(git symbolic-ref --short HEAD)
+  echo REACT_APP_VERCEL_GIT_COMMIT_SHA=$(git log --pretty=format:'%H' -n 1)
+} > .env.development.local
```

Comment:
Maybe `>>`? Or another solution to prevent accidental overwrites of other things in there?

Reply:
My thought is that we'll reserve `.env.development.local` for this one purpose in this repo, so there's no risk of overwritting anything. There are a lot of alternative `.env` file names (below) that we can use if we need more env variables for this project. (Or we can just put them in this `.sh` file if they're meant to be for local development.)

<img width="809" alt="Screen Shot 2021-12-08 at 2 53 29 PM" src="https://user-images.githubusercontent.com/2288939/145275109-cbbc2962-a2e2-4e46-881a-028ae56a8c4d.png">